content/doc/book/pipeline/jenkinsfile.adoc

```
@@ -652,64 +541,34 @@ pipeline {
 
 ==== For other credential types
 
-If you need to set credentials in a Pipeline for anything other than secret
-text, usernames and passwords, or secret files
-(<<#for-secret-text-usernames-and-passwords-and-secret-files,above>>) - i.e SSH
-keys or certificates, then use Jenkins' *Snippet Generator* feature, which you
-can access through Jenkins' classic UI.
+If you need to set credentials in a Pipeline for anything other than secret text, usernames and passwords, or secret files(<<#for-secret-text-usernames-and-passwords-and-secret-files,above>>) - i.e SSH keys or certificates, then use Jenkins' *Snippet Generator* feature, which you can access through Jenkins' classic UI.
 
 To access the *Snippet Generator* for your Pipeline project/item:
 
-. From the Jenkins home page (i.e. the Dashboard of Jenkins' classic UI), click
-  the name of your Pipeline project/item.
-. On the left, click *Pipeline Syntax* and ensure that the *Snippet Generator*
-  link is in bold at the top-left. (If not, click its link.)
-. From the *Sample Step* field, choose *withCredentials: Bind credentials to
-  variables*.
+. From the Jenkins home page (i.e. the Dashboard of Jenkins' classic UI), click the name of your Pipeline project/item.
+. On the left, click *Pipeline Syntax* and ensure that the *Snippet Generator* link is in bold at the top-left. (If not, click its link.)
```

```suggestion
. In the the left navigation pane, select *Pipeline Syntax* and ensure that the *Snippet Generator* option is available at the top of the navigation pane.
```
Suggestion to update this text to reflect the current UI better.